pkg/scheduler/scheduler.go

```diff
@@ -231,34 +237,44 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 	}
 	klog.V(3).InfoS("Attempting to schedule subscription", "subscription", klog.KObj(sub))
 
+	var finv *appsapi.FeedInventory
+	if sub.Spec.SchedulingStrategy == appsapi.DividingSchedulingStrategyType {
+		finv, err = sched.inventoryLister.FeedInventories(ns).Get(name)
+		// TODO(Garrybest): Ignore some circumstances when subscription does not need to be scheduled so far.
+		// E.g., when feed in subscription has been modified but finv has not been updated.
+		if err != nil {
+			if !errors.IsNotFound(err) {
+				utilruntime.HandleError(err)
+			}
+			return
```

Comment:
What if the corresponding `FeedInventory` is not created?

We should enqueue and re-schedule later.

Reply:
I think the `FeedInventory` event handler will add the subcription. Please check the `addAllEventHandlers` function.